otter/test/models/test_cass_models.py

```diff
@@ -106,6 +114,28 @@ def _cassandrify_data(list_of_dicts):
     return _de_identify(list_of_dicts)
 
 
+class PerformTests(SynchronousTestCase):
+    """
+    Tests for :func:`perform_cql_query` function
+    """
+
+    def test_perform_query_sync(self):
+        """
+        Calls given connection's execute
+        """
+        conn = mock.Mock(spec=CQLClient)
+        conn.execute.return_value = 'ret'
```

Comment:
Could you change this to be a defer.succeed('ret')? this test could succeed without the @deferred_performer decorator, but changing the return value to succeed('ret') should make it test that it's really dealing with deferreds correctly.